hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/migration/AddNftHistoryRangesMigrationTest.java

```diff
@@ -0,0 +1,199 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.mirror.importer.migration;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.google.common.collect.Range;
+import com.hedera.mirror.common.domain.token.NftHistory;
+import com.hedera.mirror.importer.DisableRepeatableSqlMigration;
+import com.hedera.mirror.importer.EnabledIfV1;
+import com.hedera.mirror.importer.IntegrationTest;
+import com.hedera.mirror.importer.config.Owner;
+import java.io.File;
+import java.util.ArrayList;
+import lombok.RequiredArgsConstructor;
+import lombok.SneakyThrows;
+import org.apache.commons.io.FileUtils;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.jdbc.core.JdbcOperations;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.test.context.TestPropertySource;
+
+@DisableRepeatableSqlMigration
+@EnabledIfV1
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+@Tag("migration")
+@TestPropertySource(properties = "spring.flyway.target=1.87.2")
+class AddNftHistoryRangesMigrationTest extends IntegrationTest {
+
+    private static final RowMapper<NftHistory> NFT_HISTORY_ROW_MAPPER = rowMapper(NftHistory.class);
+
+    private final @Owner JdbcOperations jdbcOperations;
```

Comment:
don't need `@Owner` since the sql migration doesn't delete rows
```suggestion
    private final JdbcOperations jdbcOperations;
```